crates/storage/db/src/implementation/mdbx/tx.rs

```diff
@@ -402,10 +406,7 @@ mod tests {
         sleep(MAX_DURATION + Duration::from_millis(100));
 
         // Transaction has not timed out.
-        assert_eq!(
-            tx.get::<tables::Transactions>(0),
-            Err(DatabaseError::Open(reth_libmdbx::Error::NotFound.into()))
-        );
+        assert_eq!(tx.get::<tables::Transactions>(0), Ok(None));
```

Comment:
I'm not sure why this should be an error? I'm assuming this is because the test did not create the table previously, whereas it does now, so I updated it accordingly.